__tests__/app.test.js

```diff
@@ -99,11 +99,101 @@ test('200: Should return objects without body property', () => {
 		.get('/api/articles')
 		.expect(200)
 		.then(({ body }) => {
-			body.forEach((article) => {
+			body.articles.forEach((article) => {
 				expect(article).not.toHaveProperty('body');
 			});
 		});
 });
+describe('GET api/articles?topic=cats / mitch', () => {
+	test('200: Should return all articles that have cats as topic', () => {
+		return request(app)
+			.get('/api/articles?topic=cats')
+			.expect(200)
+			.then(({ body }) => {
+				body.articles.forEach((article) => {
+					expect(article.topic).toBe('cats');
+				});
+			});
+	});
+	test('200: Should return all articles that have mitch as topic', () => {
+		return request(app)
+			.get('/api/articles?topic=mitch')
+			.expect(200)
+			.then(({ body }) => {
+				body.articles.forEach((article) => {
+					expect(article.topic).toBe('mitch');
+				});
+			});
+	});
+	test('400: returns if invalid topic is passed', () => {
+		return request(app)
+			.get('/api/articles?topic=banana')
+			.expect(400)
+			.then(({ body }) => {
+				expect(body.msg).toBe('Bad request');
+			});
+	});
+});
+describe('GET api/articles?sort_by', () => {
+	test('200: should return articles sorted by date in descending order by default', () => {
+		return request(app)
+			.get('/api/articles')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.articles).toBeSortedBy('created_at', { descending: true });
+			});
+	});
+	test('200: should return articles sorted by specified path in descending order', () => {
+		return request(app)
+			.get('/api/articles?sort_by=author')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.articles).toBeSortedBy('author', { descending: true });
+			});
+	});
+	test('200: should return articles sorted by specified path in descending order', () => {
+		return request(app)
+			.get('/api/articles?sort_by=title')
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.articles).toBeSortedBy('title', { descending: true });
+			});
+	});
+	test('400: returns if invalid sort_by is passed', () => {
+		return request(app)
+			.get('/api/articles?sort_by=banana')
+			.expect(400)
+			.then(({ body }) => {
+				expect(body.msg).toBe('Bad request');
+			});
+	});
+});
+describe('GET api/articles?order=ASC / DESC', () => {
+	test('200: Should return article list ordered by descending order by default', () => {
```

Comment:
Do we need this test? As we aren't passing a query we may have already covered the default behaviour on the standard endpoint with no query.